src/routes/health.routes.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+
+import { getStatusHandler } from '../controllers/health.controllers.js';
+
+const router = express.Router();
+
+/**
+ * @openapi
+ * /health-check:
+ *  get:
+ *     tags:
+ *     - Health-check
+ *     summary: Checking app availability
+ *     description: Responds if the app is up and running
+ *     responses:
+ *       200:
+ *         description: App is up and running
+ *         content:
+ *           application/json:
+ *             example:
+ *               "message": "App is up and running"
+ *               "result": "OK"
+ */
+router.get('/', getStatusHandler);
+
```

Comment:
Optional: Add one more endpoint to fetch movie by ID and handle 404 error if movie is not found

Reply:
Done!